src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -258,7 +268,7 @@ TEST(MessageHelperTestCreate,
   object[strings::cmd_icon] = 1;
   object[strings::cmd_icon][strings::value] = "10";
 
-  vis.insert(std::pair<uint32_t, smart_objects::SmartObject*>(5, &object));
+  vis.insert(CommandsMap::value_type(5, &object));
```

Comment:
@LuxoftAKutsan, because `CommandsMap` is an alias to `std::map` and `CommandsMap::value_type` is an alias to `std::pair`